README.md

```diff
@@ -1 +1,45 @@
-# fancy-title-generator
+# Fancy Fashion Product Title Generator
+
+Fancy title generator is a web application that can help you to generate a title for your fashion products.
+
+## Prerequisites
+
+Before you get started, make sure you met the following requirements:
+* Python 3.10.2
+* git
+* node.js(npm)
+
+## Installation
+
+Install the packages for recommendation service and web frontend.
+```bash
+make install
+```
+
+## Usage
+
+Run flask api server.
+```bash
+make run/api
+```
+
+Start the web application locally on port 3000.
+```bash
+make run/web
```

Comment:
Build the bundle, then serve is statically using the API package, no need in running two webservers.

Place the javascript bundle into the flask static dir, then link it from your flask template html. https://flask.palletsprojects.com/en/2.1.x/quickstart/#static-files